src/megameklab/com/ui/tabs/PreviewTab.java

```diff
@@ -59,6 +59,7 @@ public PreviewTab(EntitySource eSource) {
 	public void refresh() {
 		boolean populateTextFields = true;
 		final Entity selectedUnit = eSource.getEntity();
+		selectedUnit.recalculateTechAdvancement();
         MechView mechView = null;
```

Comment:
Does this formatting look weird to anyone else?

Reply:
Those lines were tab-indented. Fixed now.